extensions/amp-connatix-player/0.1/amp-connatix-player.js

```diff
@@ -146,22 +246,45 @@ export class AmpConnatixPlayer extends AMP.BaseElement {
     // applyFillContent so that frame covers the entire component.
     this.applyFillContent(iframe, /* replacedContent */ true);
 
+    // append child iframe for element
     element.appendChild(iframe);
     this.iframe_ = /** @type {HTMLIFrameElement} */ (iframe);
 
-    // Return a load promise for the frame so the runtime knows when the
-    // component is ready.
-    return this.loadPromise(iframe);
+    // bind to player events (playerRendered after we can send commands to player and other)
+    this.bindToPlayerCommands_();
+    // bind to amp consent and send consent info to the iframe content and propagate to player
+    this.bindToAmpConsent_();
+
+    return this.loadPromise(iframe).then(() => this.playerReadyPromise_);
+  }
+
+  /** @override */
+  isLayoutSupported(layout) {
+    return isLayoutSizeDefined(layout);
+  }
+
+  /** @override */
+  onLayoutMeasure() {
+    if (!this.iframe_) {
+      return;
+    }
+    const {width, height} = this.getLayoutBox();
+    this.sendCommand_('ampResize', {'width': width, 'height': height});
   }
 
   /** @override */
   pauseCallback() {
-    this.sendCommand_('pause');
```

Comment:
has this been renamed on the player side? If so, are you planning to continue supporting the old name for backward compatibility?

Reply:
@wassgha This was renamed in the player side. Also, I saw that it was not sent from the embed code to the player script and I renamed so, in this case, won't be breaking changes.